tests/test_rescue.py

```diff
@@ -511,6 +512,67 @@ def test_mark_for_deletion_setter_bad_types(garbage, rescue_plain_fx: Rescue):
     assert name in rat_cache_fx.by_name
 
 
+@pytest.mark.asyncio
+@pytest.mark.parametrize("uuid,name", [(uuid4(), "foo"), (uuid4(), "bar"), (uuid4(), "potato")])
+async def test_add_rat_returns_rat_by_object(uuid: uuid4, name: str, rescue_plain_fx: Rescue):
```

Comment:
invalid type hint. `uuid4` is a classmethod. `UUID` is the type.

Reply:
I copied another test (one of yours?) that has it set that way.... >.>